Pages/Index.cshtml

```diff
@@ -70,30 +70,36 @@
 }
 
 <h3 class="text-center w-100 text-dark h3 mt-5 mb-2">Safest Neighborhoods In Cincinnati</h3>
+<!-- Wrapped the content in a container class to center it on the page -->
```

Comment:
Oooooof.  Don't have a conversation in source code comments, save that for the pull request!